docs/application/native/guides/device/diagnostics.md

```diff
@@ -0,0 +1,171 @@
+
+![Diagnostics API overall architecture](media/diag-arch.png)
+
+## Prerequisites
+
+To enable your application to use Diagnostic API:
+
+1. To use Diagnostics API to collect diagnostics data, the application must be signed with a platform level distributor certificate.
+
+   To be able to sign application with a platform level certificate:
+
+   1. Open Certificate Manager in the Tizen Studio which is available in the **Tools** menu.
+   2. To add a new certificate profile, click **+** in the Certificate Manager.
+
+      ![Add a new profile](media/diag-add-new-profile.png)
+
+   3. Enter the profile name.
+
+      ![Enter the profile name](media/diag-set-profile-name.png)
+
+   4. Select **Select an existing author certificate**.
+
+      ![Select **Select an existing author certificate**](media/diag-select-author-cert.png)
+
+   5. Choose the **author certificate** from the disk using the **Browse** button.
+
+      ![Choose the author certificate](media/diag-choose-tizen-author-cert.png)
+
+   6. Select **Select a distributor certificate for an another app store**.
+
+      ![Select **Select a distributor certificate for an another app store**](media/diag-select-distributor-cert.png)
+
+   7. Choose the **distributor certificate** from the disk using the **Browse** button.
+
+      ![Choose the distributor certificate](media/diag-choose-tizen-distributor-cert.png)
+
+      This newly added profile appears in the profile list.
+
+   8. Click the **Close** button, to lose the window.
+
+      ![Newly added profile](media/diag-finish.png)
+
+2. To use functions and data types of Diagnostics API, include the `<diagnostics.h>` header file in your application:
+
+   ```cpp
+   #include <diagnostics.h>
+   ```
+
+## Crash reports
+
+Diagnostics API allows you to read the crash reports in case the process in the system fails. To do so, you must register the callback that will be called when the crash report is created:
+
+```cpp
+void crash_callback(diagnostics_ctx_h ctx, void *user_data)
+{
+    [...]
+}
+
+int register_callback(void *user_data)
+{
+    return diagnostics_set_notification_cb(crash_callback, user_data);
+}
+```
+
+> [!NOTE]
+> To deregister the callback use:
+> ```
+> diagnostics_unset_notification_cb();
+> ```
+
+In case a new crash report is created, callback is called. To read the content of the crash report use `diagnostics_get_data()`:
+
+```cpp
+void crash_callback(diagnostics_ctx_h ctx, void *user_data)
+{
+    diagnostics_data_h data;
+    const char *params[] = {"cs_info_json"};
+
+    int ret = diagnostics_get_data(ctx, params, 1, &data);
+    if (ret != DIAGNOSTICS_ERROR_NONE)
+        return;
+
+    [...]
+}
+```
+
+Using the `params` argument it is possible to specify, which data must be read from the crash report. Currently, the proper values are:
+
+-   `"cs_info_json"`: JSON report is returned, which contains information about crashed process (callstack, mapped memory regions, and so on).
```

Comment:
Would you please check below suggestion?

-   `"cs_info_json"`: JSON report is returned, which contains information about crashed process such as callstack, mapped memory regions, and so on.